### ISSUE TITLE
Using RBAC Authorization - Clarify that rules cannot be specified for Aggregated ClusterRoles

### ISSUE THREAD

**Issue by adammw** [kubernetes/website#34816]
Clarify that rules cannot be specified for Aggregated ClusterRoles as they will be overwritten. The only hint to this scenario is from the code-comment "The control plane automatically fills in the rules", it would be clearer if the fact that rules _cannot_ be specified was stated in the body text.

**Comment by jihoon-seo** [comment]
Page: https://kubernetes.io/docs/reference/access-authn-authz/rbac/
/language en

**Comment by bishal7679** [comment]
I would like to work on this issue 
Could you elaborate what exactly this issue want to be fixed?
@jihoon-seo or anyone

**Comment by shannonxtreme** [comment]
In the section [Aggregate ClusterRoles](https://kubernetes.io/docs/reference/access-authn-authz/rbac/#aggregated-clusterroles), after the sentence `The aggregationRule defines a label selector that the controller uses to match other ClusterRole objects that should be combined into the rules field of this one.`, add the following line:

```
The control plane overwrites any values that you manually specify in the `rules` field of an
aggregate ClusterRole. If you want to change or add rules, do so in the `ClusterRole` objects
that are selected by the `aggregationRule`.
```

**Comment by mehabhalodiya** [comment]
Closing this as it is being addressed in #37387

/close
